### ISSUE TITLE
Whitelist OR blacklist to which documents have spell check enabled by default

### ISSUE THREAD

**Issue by MaxMilton** [bartosz-antosik/vscode-spellright#224]
In order to spell check a document type by default it's currently it's necessary to manually add every document language type to the `spellright.documentTypes` option (a whitelist approach).

I would much rather a way to enable spell check for _all_ document language types and then use a blacklist to select which types I _don't want_ spell check to be enabled by default. Right now I have quite a long list in `spellright.documentTypes` and when I open a new doc type I get frustrated when spell check is not working only to remember that I need to add yet another entry to `spellright.documentTypes`,

**Comment by bartosz-antosik** [comment]
It is not directly related with your request, but you realize, that you can enable spelling of particular type from interface (the "eye" symbol in status bar) just by selecting the language? I make this note only because you stress the fact that you have to add the entry "manually".

**Comment by bartosz-antosik** [comment]
Of course it is persistent! Please have a look at README & look for `spellright.configurationUpdate` and `spellright.configurationScope`.

**Comment by MaxMilton** [comment]
You're totally right! Setting `"spellright.configurationScope": "user",` solved the persistence thing for me. I jump between many projects and also between a few different machines. It was frustrating because I typically don't check in the workspace config so it never persisted across machines. I _do_ however sync my user config between machines so it alleviates much of the annoyance.

I would still enjoy spell check on for all doc type by default but I'll leave it up to you if you want to implement something like that. If not, please feel free to close this issue :)
